src/include/sof/samples/audio/c_models_inc/config.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Mihai Despotovici <mihai.despotovici@nxp.com>
+ */
+
+#ifndef __CONFIG_H__
+#define __CONFIG_H__
+
+
+// RAW AUDIO DATA CONFIGURATION
+#define MS_TO_SAMPLES(SAMPLERATE, TIME) ((SAMPLERATE) * (TIME) / 1000)
+#define SIZE_FROM_STRIDE_SIZE(STRIDE, SIZE, NO_WINDOWS) ((NO_WINDOWS - 1) * STRIDE + SIZE)
+
+#define CONFIG_SAMPLERATE 16000
+#define CONFIG_NO_CHANNELS 1
+
+#define CONFIG_WINDOW_STRIDE MS_TO_SAMPLES(CONFIG_SAMPLERATE, 20)
+#define CONFIG_WINDOW_SIZE MS_TO_SAMPLES(CONFIG_SAMPLERATE, 30)
+#define CONFIG_NO_WINDOWS 49
+
+#define CONFIG_RAW_SIZE (CONFIG_NO_CHANNELS * \
+		SIZE_FROM_STRIDE_SIZE(CONFIG_WINDOW_STRIDE, \
+		CONFIG_WINDOW_SIZE, CONFIG_NO_WINDOWS))
+
+// PREPROCESSED DATA CONFIGURATION
+#define CONFIG_PREPROCESSED_HEIGHT CONFIG_NO_WINDOWS
+#define CONFIG_SPECTROGRAM_SIZE 256
+#define CONFIG_PREPROCESSED_AVGPOOL_WIDTH 6
+#define CONFIG_PREPROCESSED_WIDTH 43
+#define CONFIG_PREPROCESSED_SIZE (CONFIG_PREPROCESSED_HEIGHT * CONFIG_PREPROCESSED_WIDTH)
+
+// DSP CONFIGURATION
+#define CPU_CLOCK 600000000
```

Comment:
We'll need to change this to use a platform specific clock frequency rather than hardcode this value which subestimates on NXP side and seems to overestimate the Intel side of things.